src/cpu/minor/execute.cc

```diff
@@ -884,6 +884,47 @@ Execute::doInstCommitAccounting(MinorDynInstPtr inst)
     cpu.stats.committedInstType[inst->id.threadId]
                                [inst->staticInst->opClass()]++;
 
+    /** Add a count for every control instruction */
+
+    if (inst->staticInst->isControl())
+    {
+        if (inst->staticInst->isReturn())
+        {
```

Comment:
```suggestion
    if (inst->staticInst->isControl()) {
        if (inst->staticInst->isReturn()) {
```
brackets should be on the same line.